src/probnum/filtsmooth/optim/_stopping_criterion.py

```diff
@@ -1,8 +1,10 @@
 """Stopping criteria for iterated filtering and smoothing."""
 import numpy as np
 
+from probnum import pnm
```

Comment:
Is there a reason you import the whole `pnm` module? same in other places. It would be good to at least do it uniformly and have an agreed style that we stick to.